english/eating-walls/find-a-toilet/door/open-door.md

```diff
@@ -2,7 +2,8 @@ As you open the door, a cold hand is placed on your shoulder.
 
 In that very moment your watch sings: 12 o'clock midnight. 
 
-You jump back as you see dark black eyes.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.